### ISSUE TITLE
Scrolling down on the volume block sets volume to 0%

### ISSUE THREAD

**Issue by manvari** [vivien/i3blocks#157]
Whenever I try to reduce the volume by scrolling down on the volume block, it sets it to 0% right away. However, it does work if I use dBs instead of percentages, similarly to #136. Changing the STEP variable with 5% in the "case 4; 5;" block, as suggested there, doesn't work either.

Unlike this issue though, my i3 is up-to-date (v4.11) and so is my i3blocks (v1.4 - tried with git too). I'm using ALSA on Arch.

**Comment by manvari** [comment]
OK - I think I've finally found the issue. It turns out that some quick googling can solve 1.5 year old issues sometimes :)

Out of curiosity, I was trying to map volume buttons on my new keyboard to the proper command so that they may be usable. Google suggests using `amixer sset Master X%+` (or `-`), and trying those commands on the command line immediately caused my volume to jump respectively to 100% and 0%.

I also found odd that `amixer sset Master 50%` made my sound barely audible. After running this command, I fired up alsamixer which showed that my master volume was actually set at 24%. alsamixer and amixer thus don't behave the same way on my setup.

I looked into the amixer man page and eventually stumbled upon the following options:

`       -R     Use the raw value for evaluating the percentage representation.  This is the default mode.

       -M     Use the mapped volume for evaluating the percentage representation like alsamixer, to be more natural for human ear.
`

... and then tried `amixer -M sset Master 50%`, which indeed sets the volume at what a human ear perceives as 50%.
 
Onto i3-blocks, here is an [excerpt from the volume script] (https://github.com/vivien/i3blocks/blob/28c2b092f7d41aa723dad18b0484dca0da1cbbf5/scripts/volume#L64-L68) which describes how volume is toggled/increased/decreased:

`case $BLOCK_BUTTON in
  3) amixer -q -D $MIXER sset $SCONTROL $(capability) toggle ;;  # right click, mute/unmute
  4) amixer -q -D $MIXER sset $SCONTROL $(capability) ${STEP}+ unmute ;; # scroll up, increase
  5) amixer -q -D $MIXER sset $SCONTROL $(capability) ${STEP}- unmute ;; # scroll down, decrease
esac`

Adding the option -M to cases 4 and 5 solves the issue.

Note that [this comment](https://github.com/vivien/i3blocks/issues/136#issuecomment-244170439) from #136 describes the same issue.
